### ISSUE TITLE
json load

### ISSUE THREAD

**Issue by laura-wang** [kenshohara/3D-ResNets-PyTorch#68]
Thanks a lot for your great work!

I have a question  about the json load. 
Since the dict in python has no order, when I try to load a dataset, e.g, UCF101, using the ucf101_01.json generated from the train/testlist01.txt, the order of the dataset is not the same as the traininglist01.txt, it just shuffle it. 
Do you think it matters? 
Thanks for your answer!

**Comment by kenshohara** [comment]
I think it is not any problem.
Because the order of training data is shuffled in data_loader,
even you use the ordered format.
